### DISCUSSION TITLE
Version for Synology DSM

### DISCUSSION THREAD

**Discussion by letizido** [nekno/DV7toDV8#16] [Ideas]
It would be really helpful to have a version of this app that runs on a Synology DSM 7 NAS. Many of us store our libraries of mkv files on a Synology NAS; it would speed things up immensely if we were able to run the app or a script direct on the NAS CPU and save the time of round-tripping multi-Gig files to our Windows or Mac.

**Comment by THX723** [comment]
See issue https://github.com/nekno/DV7toDV8/issues/15. A docker version would accommodate Synology or any NAS.
